cnf-certification-test/performance/suite.go

```diff
@@ -79,6 +90,66 @@ var _ = ginkgo.Describe(common.PerformanceTestKey, func() {
 	// Scheduling related tests ends here
 })
 
+func testLimitedUseOfExecProbes(env *provider.TestEnvironment) {
+	var compliantObjects []*testhelper.ReportObject
+	var nonCompliantObjects []*testhelper.ReportObject
+	counter := 0
+	for _, put := range env.Pods {
+		for _, cut := range put.Containers {
+			if cut.LivenessProbe != nil && cut.LivenessProbe.Exec != nil {
+				counter++
+				if cut.LivenessProbe.PeriodSeconds > minExecProbePeriod {
+					tnf.ClaimFilePrintf("Container %s is using exec probes, PeriodSeconds of LivenessProbe: %s", cut,
+						cut.LivenessProbe.PeriodSeconds)
+					compliantObjects = append(compliantObjects, testhelper.NewContainerReportObject(put.Namespace, put.Name, cut.Status.ContainerID, "Liveness exec probe has a PeriodSeconds greater than 10", true))
+				} else {
+					tnf.ClaimFilePrintf("Container %s is not using of exec probes, PeriodSeconds of LivenessProbe: %s", cut,
+						cut.LivenessProbe.PeriodSeconds)
+					nonCompliantObjects = append(nonCompliantObjects, testhelper.NewContainerReportObject(put.Namespace, put.Name, cut.Status.ContainerID, "Ensure limited use of exec probes", false))
```

Comment:
"Ensure limited use of exec probes" - > "Liveness exec probe has a PeriodSeconds less than 10."
Same for startup and Readiness probes